blog.py

```diff
@@ -316,14 +340,21 @@ def _load_post(self, post):
             content,
             empty,
             0,
+            name_title_mismatch,
         ])
-    
+
     def _is_ess_empty(self, lines: List[str]) -> int:
         content = ''.join(line.strip() for line in lines)
         is_empty = re.sub(r'\*\*.+\*\*', '', content).replace('**', '') == ''
         return 1 if is_empty else 0
 
+    def _is_name_title_mismatch(self, path, title) -> int:
+        title_new = _format_title(_file_name(path).replace('-', ' '))
```

Comment:
why don't use handle `replace('-', ' ')` in `_format_title`?

Reply:
i want to return file_name only, do not want to do any formatting as later file_name will be used directly